src/features/import-projects/helpers.ts

```diff
@@ -15,10 +15,15 @@ export const updateOrCreateProject = async (
     imageField,
   } = sourConfig;
 
+  const id = `${source}-${project[idField]}`;
+
+  const dataSource = await getDataSource();
+  if (!dataSource) return;
```

Comment:
I think we should not end here silently! We can print appropriate log or even throw an error

Reply:
Done